documentation/onboard-service.md

```diff
@@ -88,6 +99,8 @@ If it is:
 - [Request](https://technical-guidance.education.gov.uk/infrastructure/monitoring/statuscake/#statuscake) a user account and an API key
 - Create a secret "STATUSCAKE-API-TOKEN" in the "inf" keyvault, with the API key as value
 
+Any secret "X" created in the *infrastructure* keyvault is available to use in terraform using [module.infrastructure_secrets.X](https://github.com/DFE-Digital/teacher-services-cloud/blob/2019-add-environment-variable-yaml-files-to-newservice-template/templates/new_service/terraform/application/secrets.tf#L1).
```

Comment:
I've replaced the 2019-add-environment-variable-yaml-files-to-newservice-template branch as it will be deleted. But the commit IDs in links are on purpose. When the link points at a line number, the content of the line my change over time and make the link invalid. Using the commit ID URL guarantees the content remains the same.